upload/common/sqs_queue.py

```diff
@@ -0,0 +1,73 @@
+import json
+
+import boto3
+from tenacity import retry, wait_fixed, stop_after_attempt
+
+from .exceptions import UploadException
+from .logging import get_logger
+
+logger = get_logger(__name__)
+
+
+class SQSQueue:
+    """ This class encapsulates logic related to adding items from an SQS (Amazon Simple Queue Service)."""
+
+    def __init__(self, queue_url, payload, error_message):
+        self.sqs = boto3.resource('sqs')
+        self.queue_url = queue_url
+        self.payload = payload
```

Comment:
Since this class is instantiated with a message, I think there are two cleaner options here:
1) rename SQSQueue to SQSQueueMessage. rename CsumSQSQueue to CsumSQSQueueMessage and DeletionSQSQueue to DeletionSQSQueueMessage as well in this case.
2) Instantiate SQSQueue with an sqs client and queue_url but pass payload directly to function so that the class is not tied to an individual message payload